mkdocs.yml

```diff
@@ -22,6 +22,7 @@ nav:
     - Changelist: 'changelist.md'
     - Deprecations: 'deprecations.md'
     - Dependencies: 'dependencies.md'
+    - 'Accumulo Migration': 'accumulo-migration.md'
```

Comment:
Does this render with or without quotes? Just wondering if the others should be like this?

Reply:
Without. It is only needed because the String has a space in it. The others don't need quotes.